cves/CVE-2015-5963.yml

```diff
@@ -290,7 +307,12 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
```

Comment:
A case could also be made that this is actually due to lacking requirements on the logout flow, but your logic here is sound as well.